src/test/java/core/basesyntax/exception/RegistrationExceptionTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.exception;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class RegistrationExceptionTest {
```

Comment:
remove this class

Reply:
not fixed: remove this test class => testing Exception is senseless test